cmd/cluster/support/status.go

```diff
@@ -58,23 +59,26 @@ func (o *statusOptions) complete(cmd *cobra.Command, args []string) error {
 }
 
 func (o *statusOptions) run() error {
-
-	clusterLimitedSupportReasons, err := getLimitedSupportReasons(o.clusterID)
+	clusterLimitedSupportReasons, count, err := getLimitedSupportReasons(o.clusterID)
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Failed to get limited support reasons: %v\n", err)
 		return err
 	}
-
 	// No reasons found, cluster is fully supported
-	if len(clusterLimitedSupportReasons) == 0 {
+	if count == 0 {
 		fmt.Printf("Cluster is fully supported\n")
 		return nil
 	}
 
 	table := printer.NewTablePrinter(os.Stdout, 20, 1, 3, ' ')
-	table.AddRow([]string{"Reason ID", "Summary", "Details"})
+	table.AddRow([]string{"Reason ID", "Summary", "Override", "Details"})
```

Comment:
As an SRE, the meaning of `Override` isn't immediately obvious to me. I have two ideas in mind that would mitigate my concern, either titling this column `Overridden (SUPPORTEX)` or explaining that override means that a specific limited support reason has been overridden in the help text around line 30.

Reply:
ah good note `Overridden (SUPPORTEX)` sounds good to me